libs/zsdl/src/zsdl.zig

```diff
@@ -899,6 +906,45 @@ pub fn setHint(name: [:0]const u8, value: [:0]const u8) bool {
 }
 extern fn SDL_SetHint(name: [*:0]const u8, value: [*:0]const u8) i32;
 
+pub const SDL_DisplayID = u32;
+
+pub const DisplayMode = DisplayMode_SDL2;
+
+pub const DisplayMode_SDL2 = extern struct {
+    format: u32,
+    w: c_int,
+    h: c_int,
+    refresh_rate: c_int,
+    driverdata: *anyopaque,
+};
+
+pub const DisplayMode_SDL3 = extern struct {
```

Comment:
```suggestion
const DisplayMode_SDL3 = extern struct {
```
This could be private for now.